tests/integration/test_npm.py

```diff
@@ -30,6 +30,18 @@
             ),
             id="npm_lockfile3_yarn_registry",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-multiple-packages.git",
+                ref="b50c11c6f49f6df8fdfc42e6925d7ac4120c3bdd",
+                packages=(
+                    {"path": "first_pkg", "type": "npm"},
+                    {"path": "second_pkg", "type": "npm"},
+                ),
```

Comment:
The [first_pkg](https://github.com/cachito-testing/npm-cachi2-multiple-packages/blob/b50c11c6f49f6df8fdfc42e6925d7ac4120c3bdd/first_pkg/package.json#L20) and [second_pkg](https://github.com/cachito-testing/npm-cachi2-multiple-packages/blob/b50c11c6f49f6df8fdfc42e6925d7ac4120c3bdd/second_pkg/package.json#L21) have exactly the same dependencies, which means cachi2 could entirely ignore one of them and this test wouldn't notice. Both need some unique dependencies

Reply:
Added more dependencies in packages and the test is e2e now. In npm cache it lists dependencies from both packages.